src/unix/core.c

```diff
@@ -50,11 +50,15 @@
 #endif
 
 #ifdef __APPLE__
+# include <crt_externs.h>
```

Comment:
This change breaks iOS (and possibly tvOS and watchOS) builds, as `crt_externs.h` is only available for macOS as of Xcode 10.3. Xcode 11 beta 5 (11M382q) includes `crt_externs.h` for iOS, tvOS, and watchOS, and presumably will at release.

Reply:
Do you want to send a pull request? iOS is not a supported platform but we accept trivial fixes.

Reply:
Opsie.